test/unit_tests/aievec_tests/bf16_tanh/testbench.cc

```diff
@@ -0,0 +1,60 @@
+#include "../common/testbench.h"
+#include "defines.h"
+#include <algorithm>
+#include <cstdint>
+#include <cstdio>
+#include <cstdlib>
+
+void dut(bfloat16 *restrict in0, bfloat16 *restrict out0);
+void dut_ref(bfloat16 *in0, bfloat16 *out0);
+
+alignas(32) bfloat16 g_in0[IN0_SIZE];
+alignas(32) bfloat16 g_out0[OUT0_SIZE];
+alignas(32) bfloat16 g_out0Ref[OUT0_SIZE];
+
+int main(int argc, char *argv[]) {
+  std::string dataDir(TO_STR(DATA_DIR));
+  srand(10);
+  std::generate(g_in0, g_in0 + IN0_SIZE,
+                [&]() { return random_bfloat16(-4, 1, 3); });
```

Comment:
So we are only exercising abs(input) < 4 ?
Also, since this is an element-wise op, there should not be any need to set the bottom 4 bits of the mantissa to 0.